### ISSUE TITLE
lerobot-rollout --use_torch_compile silently fails: torch.compile called with both mode and options

### ISSUE THREAD

**Issue by schmidal** [huggingface/lerobot#4114]
### Ticket Type

🐛 Bug Report (Something isn't working)

### Environment & System Info

```Shell
LeRobot 0.6.0, NVidia Jetson Orin NX 16, Jetson 7.2
```

### Description

Bug: In lerobot/rollout/context.py (v0.6.0), the compile block passes both mode=cfg.torch_compile_mode and options={"triton.cudagraphs": False} to torch.compile(). PyTorch forbids this combination, so every --use_torch_compile=true run logs Failed to apply torch.compile: Either mode or options can be specified, but both can't be specified at the same time and silently continues in eager mode. The RTC engine still reports "torch.compile enabled" and performs its warmup inferences, masking the failure.

Fix: Drop the options line (or the mode line). Note the current options value would also disable CUDA graphs — the main benefit of reduce-overhead on CPU-bound edge devices.
Impact (measured): SmolVLA fine-tune, Jetson Orin NX 16 GB (JetPack 7.2, torch 2.11.0+cu130, bf16), batch 1: eager 792 ms → compiled(default) 264 ms → compiled(reduce-overhead) 180 ms median. Numerical parity eager vs. compiled: cosine 0.99998.

Repro: any lerobot-rollout ... --use_torch_compile=true; observe the warning in the log.

### Relevant logs or stack trace

```Shell

```

### Checklist

- [x] I have searched existing tickets to ensure this isn't a duplicate.
- [x] I am using the latest version of the `main` branch.
- [x] I have verified this is not an environment-specific problem.

**Comment by imstevenpmwork** [comment]
Solved by https://github.com/huggingface/lerobot/issues/4114
